sys/contrib/dev/iwlwifi/iwl-trans.h

```diff
@@ -252,7 +252,7 @@ struct iwl_rx_cmd_buffer {
 
 static inline void *rxb_addr(struct iwl_rx_cmd_buffer *r)
 {
-	return (void *)((unsigned long)page_address(r->_page) + r->_offset);
+	return (char *)page_address(r->_page) + r->_offset;
```

Comment:
Could just change the unsigned long to uintptr_t to keep the diff more targeted

Reply:
Ok